### PR TITLE
POSIX shell, seriously?

### PR DESCRIPTION
**Pull request by l29ah**

### CODE REVIEW

File: tools/decode-stacktrace.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/python2
```
Comment (slaff) [inline tools/decode-stacktrace.py]: @l29ah Please, replace /usr/bin/python2 with /usr/bin/python.

Comment (avr39-ripe) [inline tools/decode-stacktrace.py]: even more proper way is to use 
#!/usr/bin/env python
invocation of python. It is more platform independent then hardcoded path

**Comment by l29ah** [comment]
On Tue, Apr 11, 2017 at 10:50:15PM -0700, slaff wrote:
> @l29ah Please, replace /usr/bin/python2 with /usr/bin/python.

/usr/bin/python is python3 on some of the modern systems, and the script is incompatible with it.

-- 
()  ascii ribbon campaign - against html mail
/\  http://arc.pasp.de/   - against proprietary attachments

**Comment by slaff** [comment]
>  and the script is incompatible with it.

I am not sure about that. Can you tell me with which python3 version did you try and it did not work?

File: tools/decode-stacktrace.py (hunk 1 above)
Comment (slaff) [inline tools/decode-stacktrace.py]: Yes, please go for "#!/usr/bin/env python".

**Comment by l29ah** [comment]
On Wed, Apr 12, 2017 at 05:36:39AM -0700, slaff wrote:
> I am not sure about that. Can you tell me with which python3 version did you try and it did not work?

3.4.6.

-- 
()  ascii ribbon campaign - against html mail
/\  http://arc.pasp.de/   - against proprietary attachments

**Comment by slaff** [comment]
> 3.4.6.

Can paste the error output here?

**Comment by slaff** [comment]
@l29ah Please, change the name of the PR to some more descriptive as "Changed Python Interpreter". Then change the first line to `#!/usr/bin/env python` and finally paste the errors that you are experiencing with Python 3.4.6 plus information about the OS and python interpreter itself.

**Comment by slaff** [comment]
This change will be merged with PR #1088.

**Pull request closed without merging**